cypress/fixtures/constants.ts

```diff
@@ -70,6 +70,59 @@ export class Constants {
         diferentReactVersionsApp1ReactBlockSubheader: 'And these are children passed into it from the legacy app',
         diferentReactVersionsParagraph: 'More react components from App2 using non-legacy React to render',
         diferentReactVersionsButtonName: 'App 2 Button',
+        comprehensiveDemoButtonTextApp5: 'bar',
+        comprehensiveDemoAlertTextApp5: 'Hello',
+        comprehensiveDemoAlertMessage: 'You have pressed a button.',
+        comprehensiveDemoHeaderTextApp4: 'Hello From Svelte world!',
+        comprehensiveDemoHeaderTextApp3: 'Styled Components App',
+        comprehensiveDemoHeaderButtonTextApp3: '💅 Test Button',
```

Comment:
constant with such text should be already added

Reply:
Fixed